.github/workflows/publish_tag.yml

```diff
@@ -2,7 +2,7 @@ name: Publish to PyPI
 on:
   push:
     tags:
-      - 'v[1-9][0-9].[0-9]+.[0-9]+'
+      - 'v[1-9][0-9].[0-9]+.[0-9].[a-z]+'
     branches: [main]
```

Comment:
Even if we want to keep two workflows, can we clean up this workflow more? Here, this workflow will still get triggered because `branches: [main]`

Reply:
Oh maybe I misunderstand how this works, but I thought this would require both the branch to be main and the tag to match. Otherwise this would have been triggered before every time we pushed to main right?

Reply:
I went ahead and removed that line